tests/integration-tests/configs/common/common.yaml

```diff
@@ -266,17 +237,13 @@ dcv:
 disable_hyperthreading:
   test_disable_hyperthreading.py::test_hit_disable_hyperthreading:
```

Comment:
The OSes not tested in the integration-test are not covered by any other test (kitchen or unit)

Reply:
I'm not able to see OS specific code related to disable_hyperthreading